### PR TITLE
add: single video page

### PR DESCRIPTION
**Pull request by Varu98**
Please use the following login credentials:-
email:- vardaan.agarwal.98@gmail.com
password:- 123

### CODE REVIEW

File: src/frontend/Components/History/History.jsx [hunk 1]
```diff
@@ -35,11 +36,13 @@ const History = () => {
...
-                  <img
-                    className="thumbnail__img"
-                    src={thumbnail_imgUrl}
-                    alt=""
-                  />
+                  <Link to={`/video/${id}`}>
+                    <img
+                      className="thumbnail__img"
+                      src={thumbnail_imgUrl}
+                      alt=""
```
Comment (vashnavichauhan1825) [inline src/frontend/Components/History/History.jsx]: provide some value to alt

File: src/frontend/Components/SingleVideoPage/SingleVideoPage.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                      alignItems: "flex-start",
+                      gap: ".5rem",
+                    }}
+                  >
+                    <h3>{video.title}</h3>
+                    <span>{video.views}</span>
+                    <div className="single-video-author">
+                      <span>
+                        <img
+                          style={{ borderRadius: "50%", width: "3rem" }}
```
Comment (vashnavichauhan1825) [inline src/frontend/Components/SingleVideoPage/SingleVideoPage.jsx:49]: try to not use inline styling

**Review by vashnavichauhan1825 — APPROVED** [review]
just few suggestion , good code structure 👍 approved for merge

File: src/frontend/Components/History/History.jsx [hunk 3]
```diff
@@ -35,11 +36,13 @@ const History = () => {
...
+                    />
+                  </Link>
                 </div>
                 <div className="card-body">
                   <img src={author_miniImgUrl} alt="" />
```
Comment (aniketsingh13) [inline src/frontend/Components/History/History.jsx]: provide some alt value here

File: src/frontend/Components/SingleVideoPage/SingleVideoPage.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
+.single-video-page__wrapper {
+  display: grid;
+  grid-template-columns: 4fr 1fr;
+  align-items: start;
+}
+.video-wrapper {
+  /* background-color: black; */
```
Comment (aniketsingh13) [inline src/frontend/Components/SingleVideoPage/SingleVideoPage.css]: remove the commented code before pushing to the main

File: src/frontend/Components/SingleVideoPage/SingleVideoPage.jsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+                          src={video.author_miniImgUrl}
+                          alt=""
```
Comment (aniketsingh13) [inline src/frontend/Components/SingleVideoPage/SingleVideoPage.jsx]: provide alt value here also

File: src/frontend/Components/VideoCard/VideoCard.jsx [hunk 6]
```diff
@@ -27,7 +28,9 @@ const VideoCard = () => {
               }}
               className="thumbnail"
             >
-              <img className="thumbnail__img" src={thumbnail_imgUrl} alt="" />
+              <Link to={`/video/${id}`}>
+                <img className="thumbnail__img" src={thumbnail_imgUrl} alt="" />
```
Comment (aniketsingh13) [inline src/frontend/Components/VideoCard/VideoCard.jsx]: here also

File: src/frontend/Components/VideoCard/VideoCard.jsx [hunk 7]
```diff
@@ -27,7 +28,9 @@ const VideoCard = () => {
...
+              </Link>
             </div>
             <div className="card-body">
               <img src={author_miniImgUrl} alt="" />
```
Comment (aniketsingh13) [inline src/frontend/Components/VideoCard/VideoCard.jsx]: here

**Review by aniketsingh13 — APPROVED** [review]
code structure look good to merge. leave some comments

**Pull request merged**